content/en/blog/hacktoberfest-2023.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Hacktoberfest 2023"
+date: 2023-09-26T16:53:13+01:00
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+Hacktoberfest 2023 is here, and we are excited to announce that Kairos is taking part by creating some good first issues
+for new contributors to get started with and also by hosting a Hacktoberfest event on October 19th in Brussels.
+
+## Why is Hacktoberfest important to us?
+
+We believe in open source, this is why all of Kairos' source code is fully open-sourced, but our commitment doesn't stop
+there, we also want to help developers, students and anyone interested to learn more about open source and how they can
+contribute to it.
+
+## Why is Kairos a good project to contribute to?
+
+Kairos is one of those projects that touches many different areas of software development, from the frontend to the
+backend. While code is what makes Kairos work, our project would not be easy to use without the documentation, and it
+would not be possible to maintain without the community.
+
+### For users
+
+If you are a user of Kairos, you can help us by reporting bugs, requesting features, or even by writing documentation.
+No matter how small you think your contribution is, it will help us make Kairos better for everyone.
+
+### For developers
+
+Whether you are an experienced developer or just getting started, Kairos has something to offer. This means you can
+contribute to the project using the skills you already have, or you can learn new ones. You don't need to be an expert
+in any of these areas to contribute, we are here to help you learn and grow. Have a look at our stack: 
+
+- Coding
+  - Go
+  - Shell scripting
+  - JavaScript
+  - Testing
+- Linux
+  - Distributions (Debian, Ubuntu, Fedora, Arch, etc)
+  - Init systems (Systemd and OpenRC)
+  - Package managers
+  - Initramfs
+  - Dracut
+  - Immutability
+- Containers and Dockerfiles management
+- Kubernetes
+- Static Site Generators
+- Edge computing
+- User Interfaces
+  - Web
+  - Command Line
```

Comment:
I'm not sure if this is the best way to present this, do you have a better idea? I just want to make it clear that there are multiple areas for learning/contributing in our project

Reply:
I think it's fine.